### ISSUE TITLE
Linear Model Tutorial: How to extract prediction? 

### ISSUE THREAD

**Issue by strman** [tensorflow/tensorflow#11440]
Similar to this issue: https://github.com/tensorflow/tensorflow/issues/97 

For the "[TensorFlow Linear Model Tutorial](https://www.tensorflow.org/tutorials/wide)", the project implies that it will end with a program that, based on input data, outputs a 0 or 1: 

> Given census data about a person such as age, gender, education and occupation (the features), we will try to predict whether or not the person earns more than 50,000 dollars a year (the target label). We will train a logistic regression model, **and given an individual's information our model will output a number between 0 and 1, which can be interpreted as the probability that the individual has an annual income of over 50,000 dollars.**

However, it seems that the tutorial is incomplete. The last steps have you calculate the accuracy of the trained model: 

> The first line of the output should be something like accuracy: 0.83557522, which means the accuracy is 83.6%. Feel free to try more features and transformations and see if you can do even better!

And then point you in the direction of the full example code: 

> If you'd like to see a working end-to-end example, you can download our [example code.](https://www.github.com/tensorflow/tensorflow/blob/r1.2/tensorflow/examples/learn/wide_n_deep_tutorial.py) and set the model_type flag to wide. 

When I run the final program, my output looks like this: 

> `accuracy: 0.989583`
> `accuracy/baseline_label_mean: 0.364583`
> `accuracy/threshold_0.500000_mean: 0.989583`
> `auc: 1.0`
> `auc_precision_recall: 1.0`
> `global_step: 3000`
> `labels/actual_label_mean: 0.364583`
> `labels/prediction_mean: 0.369466`
> `loss: 0.0242721`
> `precision/positive_threshold_0.500000_mean: 0.972222`
> `recall/positive_threshold_0.500000_mean: 1.0`

Only `accuracy` is explained in the instructions, and it doesn't seem that there are final steps to complete the tutorial: to take a set of given values, and predict `income_bracket.` Can someone provide a code example, or point to documentation on how to extract final predictions after training the model?

Thanks!

**Comment by michaelisard** [comment]
@jhseu I don't know who owns this tutorial, but it does seem as if it would be nice to add instructions on outputting the predictions.

**Comment by strman** [comment]
I have the same issue with the next tutorial as well ([Wide & Deep Learning](https://www.tensorflow.org/tutorials/wide_and_deep)), which uses the same "end to end" example code: https://github.com/tensorflow/tensorflow/blob/r1.2/tensorflow/examples/learn/wide_n_deep_tutorial.py 

The tutorial ends without a practical explanation on how to use the model. How would I take a set of given values (i.e., `"age", "workclass", "fnlwgt", "education", "education_num", "marital_status", "occupation", "relationship", "race", "gender", "capital_gain", "capital_loss", "hours_per_week", "native_country",`) and predict `income_bracket`?

**Comment by jhseu** [comment]
Adding @hengtze who owns these tutorials.

**Comment by ryutah** [comment]
I could extract prediction on this tutorials as below code.

```python
predict = m.predict(input_fn=input_fn(test_file.name, num_epochs=1, shuffle=False))

for i, p in enumerate(predict):
    print("Prediction : {}, Probablities : {}".format(p["classes"], p["probabilities"]))
```

**Comment by sudhasubramaniam** [comment]
i'm not finding m.predict function in main code. Can you please explain how this prediction code works

**Comment by blairjordan** [comment]
@ryutah The answer still isn't very clear to us newbies, so I've created a github question (with a small bounty) to get a working example of how to extract the prediction:
https://stackoverflow.com/questions/47494658/saving-and-running-wide-deep-py-model

**Comment by michaelisard** [comment]
@jhseu do you know who the current owner of the tutorial is? It would be nice to get this fixed.

**Comment by jhseu** [comment]
Neal, since you last touched the tutorial, it would be good to add .predict() examples there. Also, marked contributions welcome since the tutorial code is in on github under tensorflow/docs_src/tutorials

**Comment by karrey05** [comment]
I am using tf.contrib.learn.DNNClassifier with tf version 1.2.0, but wasn't able to extract probability with .predict().  Is there a way to output probability in addition to class labels?

**Comment by tensorflowbutler** [comment]
Please remove the assignee, as this issue is inviting external contributions. Otherwise, remove the `contributions welcome` label. Thank you.

**Comment by dushmis** [comment]
Are we getting the updated tutorial anytime soon with prediction ?

**Comment by tensorflowbutler** [comment]
Please remove the assignee, as this issue is inviting external contributions. Otherwise, remove the `contributions welcome` label. Thank you.

**Comment by imsheridan** [comment]
Added a PR #17276 for the fix.
